config/database.cr

```diff
@@ -1,5 +1,19 @@
 database_name = "avram_dev"
 
+# Adding this struct makes everything work
+#
+# Removing it causes:
+#
+# BUG: trying to upcast Dexter::Formatters::JsonLogFormatter <- Dexter::Formatters::BaseLogFormatter (Exception)
+#   from ???
+#   from ???
+#   from ???
```

Comment:
I'd like to figure this out before merging

Reply:
Actually, this is a super handy feature and I'd like to release it soon. So I'll merge with this hack in place and add an issue to remove it when the bug in Crystal is fixed https://github.com/crystal-lang/crystal/issues/7481